config/validate.go

```diff
@@ -55,7 +56,9 @@ func nameSuggestion(given string, candidates []string) string {
 	for _, candidate := range candidates {
 		dist := levenshtein.Distance(given, candidate, nil)
 		if dist < 3 {
-			return candidate
+			if candidate != given {
```

Comment:
[nit] The comparison `candidate != given` could be described succinctly using the Levenshtein distance.